Library/Homebrew/cli_parser.rb

```diff
@@ -34,9 +48,18 @@ def switch(*names, description: nil, env: nil, required_for: nil, depends_on: ni
         enable_switch(*names) if !env.nil? && !ENV["HOMEBREW_#{env.to_s.upcase}"].nil?
       end
 
+      def banner(text)
```

Comment:
Is the banner the help text? If so, a name relating to that might be more obvious.

Reply:
Banner is a brief description of the command shown above the options. 
Ex: 
```ruby
      banner <<~EOS
         Usage: brew audit [options] [<formulae>]
         Check <formulae> for Homebrew coding style violations. This should be
         run before submitting a new formula.
         If no <formulae> are provided, all of them are checked.
      EOS
```

Reply:
@GauthamGoli Ok, I see. Maybe `usage_banner` or `help_banner` or similar?